dev-client/src/screens/LocationScreens/components/soilId/SoilMatchTile.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+import {Pressable} from 'react-native';
+
+import {Icon} from 'terraso-mobile-client/components/icons/Icon';
+import {Box, Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {formatPercent} from 'terraso-mobile-client/util';
+
+type Props = {soil_name: string; score: number; onPress: () => void};
+
+export const SoilMatchTile = ({soil_name, score, onPress}: Props) => {
+  const {t} = useTranslation();
+  // TODO-cknipe: dp versus px?
+
+  return (
+    <Pressable onPress={onPress}>
+      <Box
+        backgroundColor="background.secondary"
+        borderRadius="4px"
```

Comment:
suggestion: if we are using the same border radius in many places, can we make it a constant?

Reply:
Oo ya in fact I will make `tile` a theme variant